lib/store/output.js

```diff
@@ -57,7 +57,7 @@ export function reduceOutputs(
 export function isSingeLine(text: string, availableSpace: number) {
```

Comment:
Since you have discovered a bug that allows `undefined` values to sneak in, we should change this `text: string` to a  [flow maybe type](https://flow.org/en/docs/types/maybe/), which looks like this: `text: ?string`. 

The  `?string` type will tip `flow` off that it is possible for the argument `text` to be `undefined`. Then `flow` will warn us that we need to handle the case where `text === undefined`.